src/auth/casl-ability.factory.ts

```diff
@@ -93,6 +93,7 @@ export class CaslAbilityFactory {
   }
 
   createForConsultation = async (user: User, consultationId: number) => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     const { can, cannot, build } = new AbilityBuilder<AppAbility>(
```

Comment:
`cannot` is not used at all in this function, so this warning was valid, we can remove `cannot`